Sources/ScoutCLT/Paths/PathsCommand.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Scout
+// Copyright (c) Alexis Bridoux 2020
+// MIT license, see LICENSE file for details
+
+import Foundation
+import Scout
+import ArgumentParser
+
+extension PathElementFilter.ValueType: EnumerableFlag {}
+
+struct PathsCommand: ScoutCommand {
+
+    // MARK: - Constants
+
```

Comment:
Please remove this duplicated mark.

Reply:
Fixed in latest commit 739ca313022c667ae53fc96f79a81c50d6a8264f